tests/pageObjects/AddCountryPage.js

```diff
@@ -0,0 +1,37 @@
+class AddCountryPage {
+    constructor(){
+        this.countryNameFieldSelector = "#country_name";
+        this.countryIsoNameFieldSelector = "#country_iso_name";
+        this.countryIsoFieldSelector = "#country_iso";
+        this.countryIso3FieldSelector = "#country_iso3";
```

Comment:
What is this selector for?
```suggestion
        this.countryIso3FieldSelector = "#country_iso3";
```

Reply:
This selector is for country iso 3 field. 
![image](https://github.com/user-attachments/assets/88699881-61fa-46a5-85cd-aa409f4bb7ba)
It is a three-letter [country codes](https://en.wikipedia.org/wiki/Country_code) defined in [ISO 3166-1](https://en.wikipedia.org/wiki/ISO_3166-1), part of the [ISO 3166](https://en.wikipedia.org/wiki/ISO_3166) standard published by the [International Organization for Standardization](https://en.wikipedia.org/wiki/International_Organization_for_Standardization) (ISO), to represent [countries](https://en.wikipedia.org/wiki/Country), [dependent territories](https://en.wikipedia.org/wiki/Dependent_territory), and special areas of geographical interest.